src/components/ha-hls-player.ts

```diff
@@ -186,6 +186,15 @@ class HaHLSPlayer extends LitElement {
     });
   };
 
+  private _isLLHLSSupported(): boolean {
+    // LL-HLS requires use of an http/2 proxy to avoid opening too many connections
+    return (
+      "performance" in window &&
+      performance.getEntriesByType("navigation")[0] &&
+      performance.getEntriesByType("navigation")[0].nextHopProtocol === "H2"
+    );
```

Comment:
```suggestion
    const entry =
      "performance" in window
        ? (performance.getEntriesByType(
            "navigation"
          )[0] as PerformanceNavigationTiming)
        : false;
    return entry && entry.nextHopProtocol === "H2";
```

Reply:
I took the spirit of the cast into the next iteration using PerformanceResourceTiming